### PR TITLE
Food on forks appears a bit bigger and much more centered

### PR DESCRIPTION
**Pull request by JMWTurner**
You can also attack_self() a fork with food on it to eat it.

before
before

after
after

### CODE REVIEW

**Comment by ihadtoregisterforthis** [comment]
Looks like the work of a forgotten change from 64x64.

**Comment by unid15** [comment]
Do the food's pixel_x and pixel_y really have to be there?

**Comment by clusterfack** [comment]
@ihadtoregisterforthis it has nothing to do with 64x64 genius

**Comment by ihadtoregisterforthis** [comment]
It used to look good, now it looks 1/4 of what it used to be.
And photos looked like this during 64x64 too.

**Comment by ghost** [comment]
It is to do with 64x64. I like 64x64, and I want it back, but it caused bugs that have not yet been fixed and denying that makes you look silly.

**Comment by Exxion** [comment]
It's in the wrong place to be caused by 64x64.

**Comment by ihadtoregisterforthis** [comment]
[64x64 PR](https://github.com/d3athrow/vgstation13/blame/b35eaa65d83e8e2f8fb7196044dc76872c08db98/code/game/objects/items/weapons/kitchen.dm#L121)

**Comment by Exxion** [comment]
Yes, that change multiplied a value by `PIXEL_MULTIPLIER` and did nothing else.
`PIXEL_MULTIPLIER` is currently 1.

**Comment by ComicIronic** [comment]
It looks like the issue is actually that GetFlatIcon() is causing the food sample to inherit the pixel_x and pixel_y, which displaces it on the fork.

**Comment by ghost** [comment]
It should use images, appearances, overlays and transforms instead of icons and scales.

**Comment by ghost** [comment]
Alternatively just overlays and transforms

**Comment by JMWTurner** [comment]
appearances conserve pixel offsets you know

**Comment by clusterfack** [comment]
You guys are fucking dumb if you think its caused by 64x64, especially you @DrCelt who is the last person who should be talking about bugs they caused which should be fixed.

**Comment by ghost** [comment]
@clusterfack what was it caused by, then? Which commit? I can bisect it if you want.

**Comment by ghost** [comment]
Going to check out 66d143ca3dd6348b0333253658a360264f68ee30, (before 64x64), compile, take a picture of a burger on a fork, then check out the commit after 64x64 is reverted, (78a56e1a22ba1d8d6023ed9bdae55a6de7fe8a39) (after 64x64), compile, take a picture of a burger of a fork and take a picture. Is there anything wrong with my methodology there?

**Comment by Exxion** [comment]
Make sure you test the burger at different pixel offsets in case that has an effect
Other than that, sounds good

**Comment by ghost** [comment]
What it might very well be, actually, is that the pixel_x & pixel_y offset of something placed on a table messes with it, as @Exxion suggests, but in that case, how would this PR fix it?

File: code/game/objects/items/weapons/kitchen.dm [hunk 1]
```diff
@@ -116,10 +120,10 @@
 	if(snack.reagents.total_volume)
 		loaded_food_name = snack.name
 		var/icon/food_to_load = getFlatIcon(snack)
-		food_to_load.Scale(16,16)
+		food_to_load.Scale(24 * PIXEL_MULTIPLIER,24 * PIXEL_MULTIPLIER)
 		loaded_food = image(food_to_load)
```
Comment (ghost) [inline code/game/objects/items/weapons/kitchen.dm:124]: how this works is absolutely awful by the way

**Comment by Exxion** [comment]
Actually JMW mentioned it first but whatever

**Comment by ghost** [comment]
same difference

**Comment by ghost** [comment]
Before 64x64, having checked out 66d143ca3dd6348b0333253658a360264f68ee30 - webm: http://puu.sh/razwq/ecbfc10026.webm

**Comment by ghost** [comment]
Because the video encode quality was bad, sorry about that, I'll post an image  which is the two forked burgers side by side, as shown in the video but actually at a decent quality - there is no noticeable difference (to my eyes) despite them being at practically opposite sides of the table in terms of pixel_x & pixel_y

**Comment by Exxion** [comment]
Yeah, that appears to work
Now try it again from right after 64x64 was reverted

**Comment by ghost** [comment]
Will do. Takes a lot longer to bisect than I thought, apparantely.

**Comment by ghost** [comment]
Interesting to note: the image I posted appears to be show the burger four times the size as deity's "before" picture

**Comment by ghost** [comment]
also, turns out the reason the video quality was so garbage is that OBS was downscaling it by 2x its actual size.
_does that remind you of anything?_

**Comment by ghost** [comment]
After 64x64, having checked out 78a56e1a22ba1d8d6023ed9bdae55a6de7fe8a39 (you can notice another 64x64 bug that has since been fixed in this video.):  http://puu.sh/raBsS/c1f2b02f52.webm - here is an image to save you time watching me testing different pixel_xs and pixel_ys, which we can see have no effect on it. 

We can therefore conclude 64x64 caused this bug.

**Comment by Exxion** [comment]
That we can, though I have no idea whatsoever how it happened.
Maybe like something got "reverted" that didn't get changed in the first place?

**Comment by ghost** [comment]
The actual bug causing food on forks not being centred, which also caused display case problems, is fixed in #11887

**Pull request closed without merging**